### PR TITLE
🌐 Add Spanish translation for `docs/es/docs/fastapi-people.md`

### PR DESCRIPTION
**Pull request by alejsdev**
🌐 Add Spanish translation for `docs/es/docs/fastapi-people.md`

I did some research in the official docs on GitHub and others and the official translation for merged is `fusionadas` but I think this is not very intuitive so I decided to keep the original word `merged` :sweat_smile:

### CODE REVIEW

File: docs/es/docs/fastapi-people.md [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+
+---
+
+**FastAPI** recibe mucho apoyo de la comunidad. Y quiero destacar sus contribuciones.
+
+Aquí están las personas que:
+
+* [Ayudan a otros con preguntas en GitHub](help-fastapi.md#help-others-with-questions-in-github){.internal-link target=_blank}.
+* [Crean Pull Requests](help-fastapi.md#create-a-pull-request){.internal-link target=_blank}.
+* Revisan Pull Rquests, [especialmente importante para traducciones](contributing.md#translations){.internal-link target=_blank}.
```
Comment (JavierSanchezCastro) [inline docs/es/docs/fastapi-people.md:38]: Requests*

File: docs/es/docs/fastapi-people.md [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+
+Estos usuarios son los **Principales Revisores de Traducción**. 🕵️
+
+Solo hablo algunos idiomas (y no muy bien 😅). Por lo tanto, los revisores son los que tienen el [**poder de aprobar traducciones**](contributing.md#translations){.internal-link target=_blank} de la documentación. Sin ellos, no habría documentación en varios otros idiomas.
+
+{% if people %}
+<div class="user-list user-list-center">
+{% for user in people.top_translations_reviewers[:50] %}
+
+<div class="user"><a href="{{ user.url }}" target="_blank"><div class="avatar-wrapper"><img src="{{ user.avatarUrl }}"/></div><div class="title">@{{ user.login }}</div></a> <div class="count">Reseñas: {{ user.count }}</div></div>
```
Comment (JavierSanchezCastro) [inline docs/es/docs/fastapi-people.md:186]: Pondría "revisiones" en vez de "Reseñas", quizá suena muy traducción literal

File: docs/es/docs/fastapi-people.md [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+{% endfor %}
+
+</div>
+{% endif %}
+
+## Patrocinadores
+
+EStos son los **Patrocinadores**. 😎
```
Comment (JavierSanchezCastro) [inline docs/es/docs/fastapi-people.md:194]: Estos*

Comment (alejsdev) [reply docs/es/docs/fastapi-people.md]: Cambios aplicados, gracias! :nerd_face:

File: docs/es/docs/fastapi-people.md [hunk 4]
```diff
@@ -0,0 +1,259 @@
...
+
+{% endif %}
+
+## Expertos en FastAPI
+
+Estos son los usuarios que más han estado [ayudando a otros con preguntas en GitHub](help-fastapi.md#help-others-with-questions-in-github){.internal-link target=_blank}. 🙇
+
+Ellos han demostrado ser **Expertos en FastAPI** ayudando a muchos otros. ✨
+
+!!! tip "Consejo"
```
Comment (JavierSanchezCastro) [inline docs/es/docs/fastapi-people.md:71]: image
Ahí parece que ha petado, antes del último cambio se veía bien

Comment (JavierSanchezCastro) [inline docs/es/docs/fastapi-people.md:71]: Ahí parece que ha petado, antes del último cambio se veía bien

image

**Pull request closed without merging**